dev/41c899fa-89b1-4d23-93a1-e03f6f5e88eb/device.lp

```diff
@@ -0,0 +1,19 @@
+(librepcb_device 41c899fa-89b1-4d23-93a1-e03f6f5e88eb
+ (name "SGP30")
+ (description "Gas sensor for indoor air quality applications.")
+ (keywords "gas,voc,tvoc,co2eq,sensor")
+ (author "Danilo Bargen")
+ (version "0.1")
+ (created 2019-11-08T23:24:14Z)
+ (deprecated false)
+ (category 8c508cce-7904-44b9-a300-d9cc6eb44a6d)
+ (component 2f578e3d-76e4-4d6b-b635-a3041466a377)
+ (package 888ec1db-8bc5-4b66-ac62-e8f569e02c63)
```

Comment:
The package footprint looks quite different form the one in the datasheet:
![sgp-footprint](https://user-images.githubusercontent.com/1435346/70378861-ce121f80-1925-11ea-952e-7745fae622d6.png)
Is this intended?

Reply:
Updated.